droid-config-@DEVICE@.spec.template

```diff
@@ -10,12 +10,9 @@
 # Community HW adaptations need this
 %define community_adaptation 1
 
-# Sailfish OS is considered to-scale, if in the App Grid you get 4-in-a-row icons,
-# and 2-in-a-row or 3-in-a-row app covers in the Home Screen, depending on
-# how many apps are open.
-# For 4-5.5" device screen sizes of 16:9 ratio, use this formula (hold portrait):
-# pixel_ratio = 4.5/DiagonalDisplaySizeInches * HorizontalDisplayResolution/540
-# Other screen sizes and ratios will require more trial-and-error.
+# Pixel ratio 1.0 was originally jolla phone with 245ppi, and the devices
+# should roughly have their ppi compared to that. Large displays can use
+# bigger ratio if seen fit. Values are with 0.25 increments.
 %define pixel_ratio 1.0
```

Comment:
Was the formula not useful anymore to calculate the ration?

Reply:
It's unnecessarily complicating this by involving the aspect ratio, and <5.5" displays are a minority these days too.

Reply:
OK Some further guidance would be helpful on how estimate the pixel ratio.
But anyway I agree.